static/src/javascripts/projects/common/modules/experiments/tests/remote-epic-variants.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import { setEpic } from "common/modules/commercial/contributions-service";
+import config from 'lib/config';
+
+const id = 'RemoteEpicVariants';
+
+const remoteVariant: Variant = {
+    id: 'remote',
+    test: () => setEpic(id),
+    canRun: () => true,
+};
+
+export const remoteEpicVariants: Runnable<ABTest> = {
+    id,
+    start: '2020-05-01',
+    expiry: '2020-07-01',
+    author: "Nicolas Long",
+    description: "Pseudo-test to use remote service for % of contribution epics. Expected to run as highest priority test; the canRun will then narrow the audience.",
+    audience: 1,
+    audienceOffset: 0,
+    successMeasure: "Revenue/impressions equivalent to local variants",
+    audienceCriteria: "All",
+    variants: [remoteVariant],
+    canRun: () =>
+         config.get("switches.abRemoteEpicVariants") && Math.random() < 0.01 // control test % here
+    ,
+
+    variantToRun: remoteVariant,
+    showForSensitive: true, // there is special targeting logic around this so we don't set to true here
```

Comment:
This comment seems to contradict the value! Should it be "so we _do_ set to true here"?

Reply:
oh it should be false!